### PR TITLE
feat: add hko route

### PR DESCRIPTION
**Pull request by after9**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/hko/earthquake
```

## New RSS Route Checklist / 新 RSS 路由检查表

- [x] New Route / 新的路由
- [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
- [ ] Full text / 全文获取
- [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
- [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
- [ ] Parsed / 可以解析
- [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [x] `Puppeteer`

## Note / 说明

- 新增路由暂时用于提供經香港天文台分析的全球5.0級或以上及本地有感的地震資訊。
- 后期会扩展新增新增更多其它自然公益资讯。

### CODE REVIEW

File: lib/routes/hko/app.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            const pubDate = index === array.length - 1 ? '' : item.find('td.td_region a span').text();
+            const position = index === array.length - 1 ? '' : posHtml.split('<br>')[1].replace('.', '').trim();
+            return index === array.length - 1
+                ? undefined
+                : {
+                      title: `[震級:${degree}] [地點:${position}]`,
+                      pubDate: formatDate(pubDate),
+                  };
+        })
+        .filter((item) => item !== undefined);
```
Comment (TonyRL) [inline lib/routes/hko/app.ts:60]: Unnecessary use of puppeteer. The content of the table is provided from the xml `eq_app-30d_uc.xml`

Comment (after9) [reply lib/routes/hko/app.ts]: Thanks for pointing out the issue, I have update the code again, change puppeteer to ofetch, and using the xml source now. Please review again.
Thanks!

File: lib/routes/hko/app.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    const hours = Number.parseInt(timeString.slice(8, 10), 10);
+    const minutes = Number.parseInt(timeString.slice(10, 12), 10);
+
+    // 创建一个香港时区的日期对象
+    const hongKongDate = new Date(Date.UTC(year, month, day, hours - 8, minutes)); // 香港时区为 UTC+8
+
+    // 转换为 GMT 格式
+    const gmtString = hongKongDate.toUTCString();
+    return gmtString;
+}
```
Comment (TonyRL) [inline lib/routes/hko/app.ts:70]: Please use [RSSHub's utilities](https://docs.rsshub.app/joinus/advanced/pub-date#use-utilities-class) to handle date and time.

Comment (after9) [reply lib/routes/hko/app.ts]: I updated the code and using the utilites to handle date and time now. Thanks!

**Pull request merged**